src/libs/vtools/dialogs/tools/piece/dialoginternalpath.cpp

```diff
@@ -304,183 +307,183 @@ void DialogInternalPath::ListChanged()
-            ui->plainTextEditFormulaWidthBefore->setEnabled(true);
-            ui->toolButtonExprBefore->setEnabled(true);
+            ui->beforeWidthForumla_PlainTextEdit->setEnabled(true);
+            ui->beforeExpr_ToolButton->setEnabled(true);
 
             QString w1Formula = node.GetFormulaSABefore();
             if (w1Formula != currentSeamAllowance)
             {
-                ui->pushButtonDefBefore->setEnabled(true);
+                ui->beforeDefault_PushButton->setEnabled(true);
             }
             if (w1Formula.length() > 80)// increase height if needed.
             {
                 this->DeployWidthBeforeFormulaTextEdit();
             }
             w1Formula = qApp->TrVars()->FormulaToUser(w1Formula, qApp->Settings()->GetOsSeparator());
-            ui->plainTextEditFormulaWidthBefore->setPlainText(w1Formula);
-            MoveCursorToEnd(ui->plainTextEditFormulaWidthBefore);
+            ui->beforeWidthForumla_PlainTextEdit->setPlainText(w1Formula);
+            MoveCursorToEnd(ui->beforeWidthForumla_PlainTextEdit);
 
             // Seam alowance after
-            ui->plainTextEditFormulaWidthAfter->setEnabled(true);
-            ui->toolButtonExprAfter->setEnabled(true);
+            ui->afterWidthForumla_PlainTextEdit->setEnabled(true);
+            ui->afterExpr_ToolButton->setEnabled(true);
 
             QString w2Formula = node.GetFormulaSAAfter();
             if (w2Formula != currentSeamAllowance)
             {
-                ui->pushButtonDefAfter->setEnabled(true);
+                ui->afterDefault_PushButton->setEnabled(true);
             }
             if (w2Formula.length() > 80)// increase height if needed.
             {
                 this->DeployWidthAfterFormulaTextEdit();
             }
             w2Formula = qApp->TrVars()->FormulaToUser(w2Formula, qApp->Settings()->GetOsSeparator());
-            ui->plainTextEditFormulaWidthAfter->setPlainText(w2Formula);
-            MoveCursorToEnd(ui->plainTextEditFormulaWidthAfter);
+            ui->afterWidthForumla_PlainTextEdit->setPlainText(w2Formula);
+            MoveCursorToEnd(ui->afterWidthForumla_PlainTextEdit);
 
             // Angle type
-            ui->comboBoxAngle->setEnabled(true);
-            const int index = ui->comboBoxAngle->findData(static_cast<unsigned char>(node.GetAngleType()));
+            ui->angle_ComboBox->setEnabled(true);
+            const int index = ui->angle_ComboBox->findData(static_cast<unsigned char>(node.GetAngleType()));
             if (index != -1)
             {
-                ui->comboBoxAngle->setCurrentIndex(index);
+                ui->angle_ComboBox->setCurrentIndex(index);
             }
         }
     }
     else
     {
-        ui->plainTextEditFormulaWidthBefore->setPlainText("");
-        ui->plainTextEditFormulaWidthAfter->setPlainText("");
-        ui->comboBoxAngle->setCurrentIndex(-1);
+        ui->beforeWidthForumla_PlainTextEdit->setPlainText("");
```

Comment:
beforeWidthFormula_PlanTextEdit

Reply:
beforeWidthFormula. :)

BTW... you've probably noticed by now I have a dislike for Roman's weird reversed right to left non english naming convention. All one has to do is look at Qt's own code or examples. <sigh>

Reply:
yep me too  :)